javascript/ql/test/query-tests/Security/CWE-078/lib/subLib/index.js

```diff
@@ -1,7 +1,7 @@
 var cp = require("child_process")
 
 module.exports = function (name) {
-	cp.exec("rm -rf " + name); // OK - this file belongs in a sub-"module", and is not the primary exported module.
```

Comment:
Hmm. We explicitly decided not to flag this kind of pattern earlier. Should we do a run-on-all to ensure that we are not forgetting something?

Reply:
> Hmm. We explicitly decided not to flag this kind of pattern earlier. Should we do a run-on-all to ensure that we are not forgetting something?

Lets do that.  

I think the reason I added the filter was from seeing noise in various sub-packages.  
Often these sub-packages were just copy-pasted third party libraries, or only used internally.   

But they're not FPs, they're just benign results (and people should stop copy-pasting code).  

I'm running [an evaluation](https://github.com/dsp-testing/erik-krogh-dca/tree/run/topPack-default-libaryInputs/reports) now, lets wait for that before we do a run-on-all.